### ISSUE TITLE
A-10C2s defaulting to air-start and never spawning

### ISSUE THREAD

**Issue by Biggus22** [dcs-liberation/dcs_liberation#2135]
### Affected versions

Development build

### Development build

https://github.com/dcs-liberation/dcs_liberation/actions/runs/2045067691

### Description

Noticed this in two campaigns today, Taskforce Thunder and Tripoint Hostility.
Player A-10C2s never seem to spawn, and after checking that they were cold-starts at mission planning, they seem to revert back to air starts.

I tried to set up a single flight with client four A-10C2s as the only blufor flight in one mission and they exhibited the same behavior.

### Save game and other files

[Syria-TFThunder.zip](https://github.com/dcs-liberation/dcs_liberation/files/8362583/Syria-TFThunder.zip)

**Comment by Hornet2041** [comment]
Something strange is definitely going on. I'm trying a clean tripoint hostility campaign and it doesn't even load the airplane for me when slotting into an a-10.

**Comment by Hornet2041** [comment]
The parking slots at H4 are too small for the A-10C II, but it can still do a runway start. Liberation will default it to in-flight start though.

**Comment by Biggus22** [comment]
Yes, was able to get them to cold start spawn at H3.

**Comment by DanAlbert** [comment]
This sounds like it's probably a campaign bug (shouldn't be putting planes at that airfield).

**Comment by Starfire13** [comment]
Yes, there are at least two campaigns where this is a common issue, both due to the use of sub-optimal airfields.

**Comment by DanAlbert** [comment]
Task Force Thunder doesn't seem to exist, so @sgtfuzzle17 for tripoint

**Comment by DanAlbert** [comment]
@tmz42 took over that campaign.

**Comment by tmz42** [comment]
I'll try to give Tripoint a bit of love next week.
